packages/components/src/globals/scss/_typography.scss

```diff
@@ -45,6 +45,6 @@ $base-font-size: 16px !default;
 /// @param {Bool} $fluid [false] - Specify whether to include fluid styles
 /// @example @include type-style('body-short-01');
 /// @group global-typography
-@mixin type-style($name, $fluid: false) {
-  @include carbon--type-style($name, $fluid);
+@mixin type-style($name, $tokenOverrides: $tokens) {
```

Comment:
What do we need to pass `tokenOverrides` in for? Is it unsafe for us to assume that the token we're trying to include isn't available as the current value?

Reply:
I need to revisit this. I see I removed `$fluid` as a param just to get the concept working.

Reply:
Recent commit reverts this change.